nowcasting_dataset/data_sources/pv/live.py

```diff
@@ -45,23 +45,42 @@ def get_metadata_from_database() -> pd.DataFrame:
     return pv_systems_df
 
 
-def get_pv_power_from_database(history_duration: timedelta) -> pd.DataFrame:
+def get_pv_power_from_database(
+    history_duration: timedelta, interpolate_minutes: int = 30, load_extra_minutes: int = 60
+) -> pd.DataFrame:
     """
     Get pv power from database
 
+    :param history_duration: a timedelta of how many minutes to load in the past
```

Comment:
Tiny nit: I think we're using numpy-style docstring in `nowcasting_dataset`? https://numpydoc.readthedocs.io/en/latest/format.html#parameters

Reply:
thanks